api/src/main/resources/underlays/aou_synthetic.yaml

```diff
@@ -276,6 +276,8 @@ entities:
         dataType: STRING
+criteriaConfigs: >
+  [{"type":"concept","title":"Conditions","defaultName":"Contains Conditions Codes","plugin":{"columns":[{"key":"concept_name","width":"100%","title":"Concept Name"},{"key":"concept_id","width":120,"title":"Concept ID"},{"key":"standard_concept","width":180,"title":"SourceStandard"},{"key":"vocabulary_id","width":120,"title":"Vocab"},{"key":"concept_code","width":120,"title":"Code"}],"entities":[{"name":"condition","selectable":true,"hierarchical":true}]}},{"type":"concept","title":"Procedures","defaultName":"Contains Procedures Codes","plugin":{"columns":[{"key":"concept_name","width":"100%","title":"Concept Name"},{"key":"concept_id","width":120,"title":"Concept ID"},{"key":"standard_concept","width":180,"title":"SourceStandard"},{"key":"vocabulary_id","width":120,"title":"Vocab"},{"key":"concept_code","width":120,"title":"Code"}],"entities":[{"name":"procedure","selectable":true,"hierarchical":true}]}},{"type":"concept","title":"Observations","defaultName":"Contains Observations Codes","plugin":{"columns":[{"key":"concept_name","width":"100%","title":"Concept Name"},{"key":"concept_id","width":120,"title":"Concept ID"},{"key":"standard_concept","width":180,"title":"SourceStandard"},{"key":"vocabulary_id","width":120,"title":"Vocab"},{"key":"concept_code","width":120,"title":"Code"}],"entities":[{"name":"observation","selectable":true}]}},{"type":"concept","title":"Drugs","defaultName":"Contains Drugs Codes","plugin":{"columns":[{"key":"concept_name","width":"100%","title":"Concept Name"},{"key":"concept_id","width":120,"title":"Concept ID"},{"key":"standard_concept","width":180,"title":"SourceStandard"},{"key":"vocabulary_id","width":120,"title":"Vocab"},{"key":"concept_code","width":120,"title":"Code"}],"entities":[{"name":"ingredient","selectable":true,"hierarchical":true},{"name":"brand","sourceConcepts":true,"attributes":["concept_name","concept_id","standard_concept","concept_code"],"listChildren":{"entity":"ingredient","idPath":"relationshipFilter.filter.binaryFilter.attributeValue","filter":{"relationshipFilter":{"outerVariable":"ingredient","newVariable":"brand","newEntity":"brand","filter":{"binaryFilter":{"attributeVariable":{"variable":"brand","name":"concept_id"},"operator":"EQUALS","attributeValue":{"int64Val":0}}}}}}}]}},{"type":"attribute","title":"Ethnicity","defaultName":"Contains Ethnicity Codes","plugin":{"attribute":"ethnicity_concept_id"}},{"type":"attribute","title":"Gender Identity","defaultName":"Contains Gender Identity Codes","plugin":{"attribute":"gender_concept_id"}},{"type":"attribute","title":"Race","defaultName":"Contains Race Codes","plugin":{"attribute":"race_concept_id"}},{"type":"attribute","title":"Sex Assigned at Birth","defaultName":"Contains Sex Assigned at Birth Codes","plugin":{"attribute":"sex_at_birth_concept_id"}},{"type":"attribute","title":"Year at Birth","defaultName":"Contains Year at Birth Values","plugin":{"attribute":"year_of_birth"}}]"
```

Comment:
Reformat this sensibly, since we'll need to be able to edit it.

Reply:
Whitespace is allowed in JSON so that doesn't cause an issue, does it? There are additional YAML options for how the whitespace is treated that could be helpful in that case.